index.html

```diff
@@ -7,6 +7,33 @@
   <link rel="stylesheet" href="./css/styles.css">
 </head>
 <body>
-  Starting html template. Build from here.
+    <header>
+      <div class="logo">
+        <img src="img/logo.png">
+      </div>
+      <div class="main">
+      <ul class="container-1">
+        <li class="active"><a href="#">Home</a></li>
+        <li><a href="#">Contacts</a></li>
+        <li><a href="#">Partners</a></li>
+        <li><a href="#">Invest</a></li>
+      </ul> 
+    </div>
+    <div class="title">
+      <h1>Cryptokickstart</h1>
+    </div>
+    </header>
+
+  <footer class="bottom_footer">
+      <nav class="main-navigation2">
+        <ul>
+          <li><a class="main-navigation__item" href="#">Home</a></li>
```

Comment:
Change main-navigation__item to footer-nav__item or whatever class you have used for the `<nav>` above as I've requested.